acl-lib/src/cmr/acl/core.clj

```diff
@@ -121,6 +121,10 @@
   "The cache key for the token to ingest management permission cache."
   :token-imp)
 
+(def token-smp-cache-key
+  "The cache key for the token to subscription management permission cache."
+  :token-smp)
+
 (def TOKEN_IMP_CACHE_TIME
   "The number of milliseconds token information will be cached."
   (* 5 60 1000))
```

Comment:
is 5 min standard for CMR for this kind of cache or is there a logic to the number?

Reply:
I guess through experience, 5 min can maximize the performance and  still provides that value that's not out of date.